staging/src/k8s.io/api/policy/v1beta1/types.go

```diff
@@ -84,6 +84,8 @@ type PodDisruptionBudgetStatus struct {
 // +k8s:prerelease-lifecycle-gen:introduced=1.5
 // +k8s:prerelease-lifecycle-gen:deprecated=1.22
```

Comment:
You can use this to indicate which API users should use instead 

```
// +k8s:prerelease-lifecycle-gen:replacement=policy,v1,PodDisruptionBudget
```

Reply:
Added.